app/controllers/galleries_controller.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 class GalleriesController < ApplicationController
+  helper_method :documents
```

Comment:
that was my first approach too, however it means either duplicating, somehow including or refactoring the cacheable_view.rb concern.